test/instanciate.test.js

```diff
@@ -0,0 +1,51 @@
+process.env.AWS_REGION = process.env.AWS_REGION || 'us-east-2'
```

Comment:
Did you mean `instantiate.test.js`?

https://www.yourdictionary.com/instanciate

Reply:
fixed